### DISCUSSION TITLE
[question] Pips above slider

### DISCUSSION THREAD

**Discussion by victor-baumann** [simeydotme/svelte-range-slider-pips#80] [Formatting / Styling] [answered]
Thanks for making this component! 

I'm implementing a design where the pips/labels are above instead of below the slider. Which is a bit unusual, but makes sense in the context. 

<img width="602" alt="Screen Shot 2022-09-09 at 8 02 07 PM" src="https://user-images.githubusercontent.com/48716/189462946-b2d3201a-cce7-4200-ab5b-e259954d03bd.png">

From quickly testing I seem to be able to achieve this with css only by setting negative values to the absolute position attributes of `.rangePips` and `.pipVal` then tweaking some other things as well.

So before going down this path, I just wanted to know, if I missed something in the docs and if there is a better way to do this.

**Comment by simeydotme** [comment, marked as answer]
Hi @victor-baumann , sorry for slow reply.
Yep this is absolutely the correct path to take! :) 

eg; https://simeydotme.github.io/svelte-range-slider-pips/#styling-example-for-pips
